### DISCUSSION TITLE
GitHub actions disabled due to malicious PR

### DISCUSSION THREAD

**Discussion by n3f4s** [community/community#26606] [Actions]
<p>I very recently received a malicious PR that looks like a cryptocurrency miner abuse. At first, I only closed the issue not knowing that the actions in the malicious PR would run. After realising that the malicious action was running despite the PR being closed, I’ve cancelled it and change the action policy to “Allow local action only” but now actions are disabled for my account.</p>
<p>How can I enable actions again for my account?</p>

**Comment by tqpcharlie** [comment, marked as answer]
<p>I’d contact GitHub support at <a href="https://support.github.com/contact" rel="noopener nofollow ugc">https://support.github.com/contact</a> and talk to them about this.</p>

**Comment by WhiteBlackGoose** [comment]
<p>Those who’re here because of the miner’s attacks, here’s my approach:</p>
<ol>
<li>Lock the conversation. (otherwise, the miner will reopen the PR)</li>
<li>Block the user. (so that no more PRs could be opened)</li>
<li>Cancel workflows (so that your Actions aren’t mining for someone’s wallet)</li>
<li>Report about the user (to let GH ban it sooner)<br>
I recommend this exact ordering.</li>
</ol>

**Comment by n3f4s** [comment]
<p>Thanks for your answers. I’ve contacted GitHub support.</p>
<p>In my case, I’ve also limited the PR for new accounts since I’ve had another similar PR, a few hours later, from another throwaway account.</p>
